.eslintrc.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
     "@typescript-eslint/explicit-function-return-type": "off",
     "react/prop-types": "off",
     "camelcase": "off",
-    "@typescript-eslint/camelcase": "off"
+    "@typescript-eslint/camelcase": "off",
```

Comment:
Where are these rules coming from? 

Ultimately we want to unify with sdk codegen and components

Reply:
Brought in the rules from sdk-codegen.

I couldn't figure out why these don't have a definition file defined:
"array-callback-return": "off",
"node/no-path-concat": "off",
"node/no-new-require": "off",
"node/handle-callback-err": "off",

So I disabled them.

And rules like this are to get around d3:
"@typescript-eslint/unbound-method": "off",
"@typescript-eslint/no-empty-function": "warn",

There is work to be done to get 1:1 parity, but this is pretty close now